packages/core/src/lib/util.ts

```diff
@@ -25,6 +25,10 @@ import {WebManifestIcon} from './types/WebManifest';
 const execPromise = promisify(exec);
 const extractZipPromise = promisify(extractZip);
 
+// Regex for disallowed characters on Android Packages, as per
+// https://developer.android.com/guide/topics/manifest/manifest-element.html#package
+const DISALLOWED_ANDROID_PACKAGE_CHARS_REGEX = /[^ a-zA-Z0-9_\.]/g;
```

Comment:
I don't think there should be a space in here.

Reply:
Yes, that shouldn't be there. Removed and added a test for input with spaces.